src/App/App.jsx

```diff
@@ -1,22 +1,29 @@
 import React from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
+import { Switch } from "react-router";
 import Login from "../Login/Login";
 import Home from "../Home/Home";
 import { AuthProvider } from "../Auth";
 import PrivateRoute from "../PrivateRoute";
 import CreateRecipe from "../CreateRecipe/CreateRecipe";
 import Registration from "../Registration/Registration";
+import Navbar from "../NavBar/NavBar";
 import Likes from "../Likes/Likes";
 
 function App() {
   return (
     <AuthProvider>
       <Router>
-        <PrivateRoute exact path="/" component={Home} />
-        <PrivateRoute exact path="/add-recipe" component={CreateRecipe} />
-        <PrivateRoute exact path="/likes" component={Likes} />
-        <Route exact path="/login" component={Login} />
-        <Route exact path="/registration" component={Registration} />
+        <Switch>
+          <Route exact path="/login" component={Login} />
+          <Route exact path="/registration" component={Registration} />
+          <div>
```

Comment:
Console has a warning. Probably something to do with this <div>. Only use divs when there is a purpose for them, otherwise use fragments (i.e. <>)

Reply:
![image](https://user-images.githubusercontent.com/33896659/119114366-66bb3300-ba26-11eb-958b-537e34905d50.png)
The warning